my-app/src/index.module.scss

```diff
@@ -1,14 +1,18 @@
-// body {
-//   margin: 0;
-//   padding: 0;
-//   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-//     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-//     sans-serif;
-//   -webkit-font-smoothing: antialiased;
-//   -moz-osx-font-smoothing: grayscale;
-// }
+// @import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap");
```

Comment:
If this has bee commented out then what font style is being used?

Reply:
Sans-serif

Reply:
Why not open-san though @Excelsior2021 ?

Reply:
I would prefer to have the fonts in the input fields `open-san`

Reply:
Everything looks really good though 🚀

Reply:
Also should you describe the changes that you did here @Excelsior2021 ?

<img width="906" alt="Screenshot 2022-09-19 at 15 09 54" src="https://user-images.githubusercontent.com/37377831/191037439-95e36983-bb6e-438f-9eae-95ae23a2f312.png">